src/Perlang.Tests.Integration/Stdlib/Base64EncodeTests.cs

```diff
@@ -45,12 +45,12 @@ public void Base64_encode_with_a_long_string_argument_returns_the_expected_resul
         [Fact]
         public void Base64_encode_with_a_numeric_argument_throws_the_expected_exception()
         {
-            var result = EvalWithValidationErrorCatch("Base64.encode(123.45)");
+            var result = EvalWithValidationErrorCatch("Base64.encode(123)");
             var runtimeError = result.Errors.First();
 
             Assert.Single(result.Errors);
 
-            Assert.Equal("Cannot pass double argument as string parameter to encode()", runtimeError.Message);
+            Assert.Equal("Cannot pass int argument as string parameter to encode()", runtimeError.Message);
```

Comment:
Likewise, using `int` here is simpler since we don't need to demonstrate the locale-dependent bugs in this context. (they are already tested elsewhere)